web/js/site.js

```diff
@@ -17,7 +17,7 @@ $(document).ready(function() {
 		cache: false,
 
 		// A user needs to change this value to get their site to work.
-		url: 'https://raw.githubusercontent.com/nirzari/review-rot/master/web/js/default-data.json',
+		url: 'https://exd.pages.redhat.com/lholecek/rhelwf-reviews/web/js/default-data.json',
```

Comment:
This doesn't work for me.
I takes me to an internal (RH) gitlab login page.

Reply:
I added this change accidentally. I will revert it.